DATA/SConscript

```diff
@@ -128,12 +128,12 @@ def getCertifyCmd(stage):
     """
     calibName = stage
     # 'gain' corresponds to 'gain from flat pairs' method.
-    # These gain estimates are stored in the dataset 'cpPtcExtract'
-    # (formerly known as 'cpCovariances').
+    # These gain estimates are stored in the dataset 'cpPtcPartial'
+    # (formerly known as 'cpPtcExtract' or 'cpCovariances').
     if stage == 'gain':
-        calibName = 'cpPtcExtract'
+        calibName = 'cpPtcPartial'
     if stage == 'cti':
-        calibName = 'cpCtiCalib'
+        calibName = 'cti'
```

Comment:
This case can be dropped, as `calibName` is already equal to `stage` which is `'cti'`.